update.sh

```diff
@@ -8,6 +8,7 @@ if [ ${#versions[@]} -eq 0 ]; then
 	versions=( */ )
 fi
 versions=( "${versions[@]%/}" )
+shaPage=$(curl -s 'https://www.ruby-lang.org/en/downloads/')
```

Comment:
https://github.com/docker-library/official-images/issues/628 is definitely relevant here -- just `curl -s` will fail silently and without even an error message if there's any trouble